src/lib/create-config.ts

```diff
@@ -182,17 +183,53 @@ const dateHandlingCallback = (connection, callback) => {
     });
 };
 
+const databaseSsl = () => {
+    if (process.env.DATABASE_SSL != null) {
+        return JSON.parse(process.env.DATABASE_SSL);
+    } else if (process.env.DATABASE_SSL_CA_CONFIG != null) {
+        return readFileSync(process.env.DATABASE_SSL_CA_CONFIG).toJSON();
+    } else if (
+        process.env.DATABASE_SSL_KEY_FILE != null &&
```

Comment:
Why is the DATABASE_SSL_KEY_FILE mandatory? For example, AWS provides only a CA file without a KEY file.